backend/src/services/spotify.ts

```diff
@@ -1,6 +1,4 @@
 import SpotifyWebApi from "spotify-web-api-node";
```

Comment:
Question: Should we switch out the backend "spotify-web-api-node" package for "spotify-web-api-ts-sdk" so we are using the same thing? https://github.com/spotify/spotify-web-api-ts-sdk